ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -251,3 +251,36 @@ VLV:
   implements:
   - EQUIPMENT
 
+RSR:
+  description: "A generic distribution riser."
+  is_abstract: true
+
+RARSR:
+  description: "A return air riser."
+  is_abstract: true
+  implements:
+  - RSR
+
+FARSR:
+  description: "A fresh air (ventilation) riser."
+  is_abstract: true
+  implements:
+  - RSR
+
+CHWRSR:
+  description: "A chilled water distribution riser."
+  is_abstract: true
+  implements:
+  - RSR
+  - CHWS
```

Comment:
Implementing CHWS seems incorrect.  I think what you mean is that it should be part of a CHWS (which we cannot represent until we implement connection constraints)